dashboard/src-tauri/src/main.rs

```diff
@@ -81,7 +103,7 @@ async fn main() {
 
     let tray_menu = SystemTrayMenu::new().add_item(quit);
 
-    builder
+    if let Err(_) = builder
```

Comment:
display error message in the notification body

Reply:
added. this one isn't our Error though so not 100% sure if it will be pretty